### ISSUE TITLE
Sprints: What we want our project management cadence to be

### ISSUE THREAD

**Issue by RichardLitt** [ipfs/pm#126]
#### What

We need to figure out how we are doing our sprints for the next three months. We can do long sprints (half a day on Monday, with lots of video calls on different topics); short sprints (a short, general video call); or something in between. 
#### Background

Around a month ago, we stopped doing our longer sprints. Before, our sprints were in two formats: long, which meant an IRC check in with tasks, followed by two or three hours of video calls, covering apps on ipfs, js-ipfs, libp2p, go-ipfs, infrastructure, and any pop up discussions people wanted to discuss, generally in thirty minute discussions. Our short sprints were a modified version: instead of talking about what we've done on IRC, we started having an hour long call where everyone on the core team would say what they were working on, what they were blocked on, and what they planned to do. 

Now that Q3 has started, we should plan out how we are going to do sprints this quarter. 

An issue was created last month, [here](https://github.com/ipfs/pm/issues/115), about whether or not we should have one-week sprints, or two week sprints. This is also relevant: the short sprint happened generally on the off-Monday's between a two week sprint, while the long sprints happened when a new sprint was decided. Going forward, we need to figure out if we want to have two week sprints again, too. 
#### Relevant Reading
- [How we do our Sprints, in the README](https://github.com/ipfs/pm/blob/master/README.md)
- [Sprint change to once every two weeks](https://github.com/ipfs/pm/issues/96)
- [Stopping the IRC copy-pasta syncs](https://github.com/ipfs/pm/issues/95)
- [The last sprint thread](https://github.com/ipfs/pm/issues/123), as an example of an empty sprint that should've been more filled out.

**Comment by RichardLitt** [comment]
This should also start, today. In two hours, we have an IRC sync scheduled on the Calendar, as well as video calls set afterwards. I believe that is enough time for us to discuss how we want today's sprint to go.

**Comment by dignifiedquire** [comment]
## Suggestion for today

1. Everyone posts their done todos and new todos in the respective Github issues.
2. Have the short sync style hangout
3. Have the video syncs where the leads feel the need for them to happen. I know that @diasdavid and I want to talk about some things with the community in js-ipfs, and there is an IPLD talk which was supposed to happen after the hangouts.

**Comment by jbenet** [comment]
I've a lot more to say on all this, but here is one tidbit:

I think a major mismatch on expectations about the sprint is this: there is a sense that the sprint is to broadcast a news report on what's been happening in the project. This is NOT the goal of the sprint calls. It may be what some people thought it was, but it's not the goal of this huge sync time expenditure. The newsletter and blog are for that purpose, and they serve much better, as consuming text is much easier. 

The sprint calls are for project developers to have a structured development cadence -- figure out what was completed in the last unit of time, what wasn't, why not, bring up goals for the next week, allocate who is doing what, etc. (a nice bonus is to have some f2f time all together, but this is strictly a secondary goal, and can be had in other ways). In recent weeks, discussion on project management sounds like people want to move to more kanban style "go as fast as we can" and away from agile/scrum sprints. this isnt fully decided yet, more discussions have to happen, but those discussions will determine what we do with sprint calls.

Let's re-frame the discussion from "long vs short" into "what we want our project management cadence to be". As such, until we get a clear picture on how we're doing roadmaps, boards, milestones, etc, we cant decide what we want to use these days for. 

I think that for today, we should use the allotted time to go over and discuss Q3 roadmap and goals for each of the main projects.

**Comment by diasdavid** [comment]
We've had the opportunity to go over this discussion several times, through hangouts, issues and in person. Now we are all committed to Weekly sprints and quarter roadmap reviews. 

Would you say that this issue has been solved?

**Comment by RichardLitt** [comment]
No longer relevant.
